api/src/main/java/org/openmrs/module/reporting/cohort/definition/library/BuiltInCohortDefinitionLibrary.java

```diff
@@ -150,4 +152,23 @@ public CohortDefinition getPersonWithAttribute() {
         cd.addParameter(new Parameter("values", "reporting.parameter.values", String.class, List.class, null));
         return cd;
     }
+
+    @DocumentedDefinition("programEnrollment")
+    public CohortDefinition getProramEnrollment(){
+        ProgramEnrollmentCohortDefinition cd = new ProgramEnrollmentCohortDefinition();
+        cd.addParameter(new Parameter("programs", "reporting.parameters.programs", Program.class, List.class, null));
+        cd.addParameter(new Parameter("enrolledOnOrAfter", "reporting.parameters.startDate", Date.class));
+        cd.addParameter(new Parameter("enrolledOnOrBefore", "reporting.parameters.endDate", Date.class));
+        cd.addParameter(new Parameter("completedOnOrAfter", "reporting.parameters.startDate", Date.class));
+        cd.addParameter(new Parameter("completedOnOrBefore", "reporting.parameters.endDate", Date.class));
+        cd.addParameter(new Parameter("locationList", "reporting.parameters.programs", Location.class, List.class, null));
```

Comment:
The labels of the above date parameters are not correct, since you are not exposing them as startDate and endDate